packages/neuron-ui/src/components/Receive/index.tsx

```diff
@@ -51,21 +41,30 @@ const Receive = ({ address }: { address: string }) => {
     // TODO: generate new address
     return '0x0da2fe99fe549e082d4ed483c2e968a89ea8d11aabf5d79e5cbf06522de6e674'
   }
+
   const accountAddress = address === undefined ? generateNewAddress() : address
+  const copyAddress = () => {
+    window.clipboard.writeText(accountAddress)
+  }
+
   return (
     <Container>
       <Card>
         <Card.Header>
           <h2>{t('Receive')}</h2>
         </Card.Header>
         <Card.Body>
-          <AddressTip>{t('Address')}</AddressTip>
-          <AddressPanel>
-            <Address onClick={() => window.clipboard.writeText(accountAddress)}>{accountAddress}</Address>
-            <OverlayTrigger placement="bottom" overlay={<Tooltip id="address-tooltip">{t('Copy address')}</Tooltip>}>
-              <CopyImage onClick={() => window.clipboard.writeText(accountAddress)} />
-            </OverlayTrigger>
-          </AddressPanel>
+          <Form.Group>
+            <Form.Label>{t('Address')}</Form.Label>
+            <AddressPanel>
+              <OverlayTrigger placement="bottom" overlay={<Tooltip id="address-tooltip">{t('Copy address')}</Tooltip>}>
```

Comment:
Please make sure we're using namespace i18n keys. (Can be in a future PR to make this one get merged soon.)

Reply:
OK

Reply:
Neuron-ui don't use namespace i18n keys now and I will create a new issue and branch to solve this question.

Reply:
yes,I am doing it